skglm/datafits/single_task.py

```diff
@@ -169,6 +172,9 @@ def gradient_scalar_sparse(self, X_data, X_indptr, X_indices, y, Xw, j):
             grad -= X_data[i] * y[idx_i] * sigmoid(- y[idx_i] * Xw[idx_i])
         return grad / len(Xw)
 
+    def intercept_update_step(self, y, Xw):
```

Comment:
We resolved to not implement intercept for logistic regression. So, I think no need for this method.
We might mark it as to do for later PR

Reply:
it does not hurt to keep it, otherwise we have to redo the work in the follow-up PR